src/modules/reporting/components/reporting-report/reporting-report.styles.less

```diff
@@ -4,3 +4,40 @@
   color: @color-lightgray;
   margin-top: 5rem;
 }
+
+.ReportingReportMarket__details {
```

Comment:
I would consider moving this out to another BEM namespace. Not a deal breaker but could help with encapsulating styling.

Reply:
The styling is for content in the Reporting Report component so I'd like the keep the styling in this same file.